cogs/guess.py

```diff
@@ -0,0 +1,97 @@
+"""importsss (that i don't know if it belongs here)
+
+    you can get rid of excess comments"""
+import random
+import asyncio
+from discord.ext import commands
+
+class Guess: # pylint: disable=too-few-public-methods
+    # fekkin pylint cogs/guess.py:9:19: C0303: Trailing whitespace (trailing-whitespace)
+    # E0012: Bad option value 'too-few-public-methods fekkin pylint' (bad-option-value)
+    # cogs/guess.py:8:0: R0903: Too few public methods (1/2) (too-few-public-methods)
+    '''this is guessing game'''
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="guess", pass_context=True)
+    async def mycom(self, context):
+        '''pylint told me to do this, this will start a guessing game :aquaThumbsUp:'''
```

Comment:
Let's be more descriptive here, and no emojis, please.  Also, let's be consistent.